doc/nrf/known_issues.rst

```diff
@@ -555,7 +564,7 @@ NCSDK-8304: HID configurator issues for peripherals connected over Bluetooth® L
 Pelion
 ======
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```

Comment:
@emob-nordic the ticket is closed with a linked issue, should this be removed?

Reply:
It should be removed, agreed. @greg-fer please do not add `v1-7-0` tag here.